### PR TITLE
point users to the jenkins tutorial from the template

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@jwforres ptal

**Comment by bparees** [comment]
@jwforres updated.

**Comment by jwforres** [comment]
LGTM

**Comment by bparees** [comment]
[merge]

File: examples/jenkins/jenkins-ephemeral-template.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "name": "jenkins-ephemeral",
     "creationTimestamp": null,
     "annotations": {
-      "description": "Jenkins service, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
+      "description": "Jenkins service, without persistent storage.\nThe username is 'admin' and the tutorial at https://github.com/openshift/origin/blob/master/examples/jenkins/README.md contains more information about using this template.\nWARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
```
Comment (spadgett) [inline examples/jenkins/jenkins-ephemeral-template.json]: @bparees I think newlines in JSON need two backslashes like `\\n`.

**Comment by spadgett** [comment]
Disregard my comment.

**Pull request merged**